product_pricelist_supplierinfo/models/product_pricelist.py

```diff
@@ -5,32 +5,6 @@
 from odoo import fields, models
 
 
-class ProductPricelist(models.Model):
-    _inherit = "product.pricelist"
-
-    def _compute_price_rule(self, products, qty, uom=None, date=False, **kwargs):
-        """Recompute price after calling the atomic super method for
-        getting proper prices when based on supplier info.
-        """
-        rule_obj = self.env["product.pricelist.item"]
-        result = super()._compute_price_rule(products, qty, uom, date, **kwargs)
-        # Make sure all rule records are fetched at once at put in cache
-        rule_obj.browse(x[1] for x in result.values()).mapped("price_discount")
-        context = self.env.context
-        for product in products:
-            rule = rule_obj.browse(result[product.id][1])
-            if rule.compute_price == "formula" and rule.base == "supplierinfo":
-                result[product.id] = (
-                    product.sudo()._get_supplierinfo_pricelist_price(
-                        rule,
-                        date=date or context.get("date", fields.Date.today()),
-                        quantity=qty,
-                    ),
-                    rule.id,
-                )
-        return result
-
-
 class ProductPricelistItem(models.Model):
```

Comment:
Since now the only model included here is `product.pricelist.item`, we should rename the file accordingly